recipes/libuv/all/test_package/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::libuv)
```

Comment:
There is an official config file since 1.44.0, it should be modeled in `package_info()` and used in test package: https://github.com/libuv/libuv/blob/v1.44.0/CMakeLists.txt#L676-L682

file name: `libuvConfig.cmake`
targets: `uv` is shared else `uv_a`